specs/referrer-policy/index.src.html

```diff
@@ -516,18 +516,18 @@ <h3 id="referrer-policy-delivery-meta">Delivery via <{meta}></h3>
   </dl>
 
   <h3 id="referrer-policy-delivery-referrer-attribute">Delivery
-  via a <code>referrer</code> attribute</h3>
+  via a <code>referrerpolicy</code> attribute</h3>
 
-  A referrer policy may be set when a <code>referrer</code> attribute is
+  A referrer policy may be set when a <code>referrerpolicy</code> attribute is
   added to an <code><a element>a</a></code>, <code><a element>area</a></code>,
   <code><a element>img</a></code> or <code><a element>iframe</a></code>
   element, for example:
 
   <pre class="example">
-    &lt;a href="http://example.com" referrer="origin"&gt;
+    &lt;a href="http://example.com" referrerpolicy="origin"&gt;
   </pre>
 
-  The following values for the <code>referrer</code> attribute are valid,
+  The following values for the <code>referrerpolicy</code> attribute are valid,
   and map to the listed <a>referrer policy</a> values:
```

Comment:
In https://fetch.spec.whatwg.org/#referrerpolicy I defined a default "" value. I think we should name the IDL attribute `referrerPolicy` btw. It's normal for IDL attributes to have proper casing.